pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -104,6 +127,19 @@ func TestClusterWideMode(t *testing.T) {
 						"expected SMMR to show member-0 as member namespace, but that wasn't the case"))
 			})
 
+			t.LogStep("Check the use of multiple selector at the same time")
+			oc.ApplyString(t, meshNamespace, smmrMultipleSelectors)
+			oc.WaitSMMRReady(t, meshNamespace)
+
+			t.LogStep("Check whether the SMMR shows only namespaces as members: member-0")
+			retry.UntilSuccess(t, func(t test.TestHelper) {
+				shell.Execute(t,
+					fmt.Sprintf("oc -n %s get smmr default", meshNamespace),
+					require.OutputContains("1/1",
+						"one namespace are in members",
+						"expected SMMR to show 1 member namespace, but that wasn't the case"))
```

Comment:
Why not check the actual namespace name instead of just the namespace count? The operator could add a completely different namespace here and the test would still pass.

Reply:
BTW, why do we create 50 member namespaces? Seems unnecessary. 5 would be more than enough.

Reply:
Improved this